FWCore/PythonUtilities/python/LumiList.py

```diff
@@ -73,12 +73,12 @@ def __init__(self, filename = None, lumis = None, runsAndLumis = None, runs = No
         if isinstance(runsAndLumis, list):
             queued = {}
             for runLumiList in runsAndLumis:
-                for run, lumis in runLumiList.items():
+                for run, lumis in list(runLumiList.items()):
                     queued.setdefault(run, []).extend(lumis)
             runsAndLumis = queued
 
         if runsAndLumis:
-            for run in runsAndLumis.keys():
+            for run in list(runsAndLumis.keys()):
```

Comment:
This would be better with the generator as well
```suggestion
            for run in runsAndLumis.keys():
```